pkg/tfgen/generate_python.go

```diff
@@ -711,8 +712,20 @@ func (g *pythonGenerator) emitPackageMetadata(pack *pkg) error {
 	w.Writefmtln("class InstallPluginCommand(install):")
 	w.Writefmtln("    def run(self):")
 	w.Writefmtln("        install.run(self)")
-	w.Writefmtln("        check_call(['pulumi', 'plugin', 'install', 'resource', '%s', '${PLUGIN_VERSION}'])",
+	w.Writefmtln("        try:")
+	w.Writefmtln("            check_call(['pulumi', 'plugin', 'install', 'resource', '%s', '${PLUGIN_VERSION}'])",
 		pack.name)
+	w.Writefmtln("        except OSError as error:")
+	w.Writefmtln("            if error.errno == errno.ENOENT:")
+	w.Writefmtln("                print(\"\"\"")
+	w.Writefmtln("                There was an issue installing the resource provider plugin.")
+	w.Writefmtln("                It looks like `pulumi` is not installed on your system.")
+	w.Writefmtln("                Please visit https://pulumi.io/ to get Pulumi.")
+	w.Writefmtln("                You may try to manually installing the plugin by running")
+	w.Writefmtln("                `plugin install resource %s ${PLUGIN_VERSION}`", pack.name)
```

Comment:
This should be `pulumi plugin install`, no?

Reply:
good catch!